speech/api/README.md

```diff
@@ -48,15 +48,17 @@ Before running these samples perform the steps:
     cd python-docs-samples/speech/api
     ```
 
-* Create virtualenv
+* Create [virtualenv][virtualenv]
     ```
     virtualenv env
     source env/bin/activate
     ```
 
 ### Install the dependencies
 
-The sample uses the [PyAudio][pyaudio] library to stream audio from your computer's microphone.  PyAudio depends on [PortAudio][portaudio], which may need to be compiled when you install PyAudio. If you run into compilation issues that mention PortAudio, you may have to [install some dependencies][pyaudio-install].
+The sample uses the [PyAudio][pyaudio] library to stream audio from your computer's microphone.  PyAudio depends on [PortAudio][portaudio], which may need to be compiled when you install PyAudio. 
+
+* You may run into compilation issues when doing pip install below, in which case [install some dependencies][pyaudio-install].
```

Comment:
...when doing the `pip install` below, in which case you may need to [install some dependencies][pyaudio-install].